scripts/hostcfgd

```diff
@@ -1405,6 +1409,35 @@ class DeviceMetaCfg(object):
                 return
             run_cmd(['sudo', 'monit', 'reload'])
 
+    def timezone_update(self, data):
+        """
+        Apply timezone handler.
+        Run the following command in Linux: timedatectl set-timezone <timezone>
+        Args:
+            data: Read table's key's data.
+        """
+        new_timezone = data.get('timezone')
+        syslog.syslog(syslog.LOG_DEBUG,
+                      f'DeviceMetaCfg: timezone update to {new_timezone}')
+
+        if new_timezone is None:
+            syslog.syslog(syslog.LOG_ERR,
```

Comment:
This check mandates timezone config to be present in DEVICE_METADATA, else would throw error. I don't think its correct. It should be optional field.